docs/content/configuration/policy-resource.md

```diff
@@ -262,6 +262,52 @@ ingressMTLS:
   verifyDepth: 1
 ```
 
+Below is an example of the `ingress-mtls-secret` using the secret type `nginx.org/ca`
+```yaml
+kind: Secret
+metadata:
+  name: ingress-mtls-secret
+apiVersion: v1
+type: nginx.org/ca
+data:
+  ca.crt: <base64encoded-certificate>
+```
+#### Using a Certificate Revocation List
+The IngressMTLS policy supports configuring at CRL for your policy.
+This can be done in one of two ways.
+
+> Note: Only one of these configurations options can be used at a time.
+
+1. Adding the `ca.crl` field to the `nginx.org/ca` secret type, which accepts a base64 encoded certificate revocation list (crl).
+Example Yaml:
+```yaml
+kind: Secret
+metadata:
+  name: ingress-mtls-secret
+apiVersion: v1
+type: nginx.org/ca
+data:
+  ca.crt: <base64encoded-certificate>
+  ca.crl: <base64encoded-crl>
+```
+
+2. Adding the `Crl` field to your IngressMTLS policy spec with the name of the CRL.
+Example Yaml:
+```yaml
+apiVersion: k8s.nginx.org/v1
+kind: Policy
+metadata:
+  name: ingress-mtls-policy
+spec:
+ingressMTLS:
+    clientCertSecret: ingress-mtls-secret
+    crl: webapp.crl
```

Comment:
Can you help me understand the expected this?

I'm assuming it is expected that the user would possibly bake this file into their container image at `/etc/nginx/secrets/webapp.crl`, or possibly mount a volume to this path. Is there some other way they may achieve this?

Is the use-case for CRLs that are larger than the 1MiB limit for Secrets?

Does this have a limitation of not automatically reloading NGINX when the file changes (whereas the `ca.crl` Secret data does trigger reload when changed)?

Adding some clarity to the documentation around these points could be helpful.

Reply:
When using the crl field in the policy we are expecting that the user is providing this file to the Ingress Controller in some way, either through a volume or their container image. There is a Note on Line 309 about this.

> Is the use-case for CRLs that are larger than the 1MiB limit for Secrets?
Yes this is the exact use case for this. I can expand the documentation to highlight this, it's not very obvious.

> Does this have a limitation of not automatically reloading NGINX when the file changes (whereas the ca.crl Secret data does trigger reload when changed)?
Yes this is a very good point. I'll make sure this is highlighted in the document. NGINX will not reload automatically when the file contents are changed. You will need to trigger a reload to pick up the latest CRL.

Reply:
> es this is a very good point. I'll make sure this is highlighted in the document. NGINX will not reload automatically when the file contents are changed. You will need to trigger a reload to pick up the latest CRL.

I believe that if you are using NGINX Plus you should not have to worry about reloading.  (I might be incorrect on that, but I think so)
https://www.nginx.com/blog/nginx-plus-r18-released/#dynamic-certificate-loading

Reply:
Thanks - those doc updates look great 👍